corehq/apps/tzmigration/timezonemigration.py

```diff
@@ -0,0 +1,190 @@
+import collections
+from copy import deepcopy
+import os
+
+from couchdbkit import ResourceNotFound
+from django.conf import settings
+from casexml.apps.case.cleanup import rebuild_case_from_actions
+from casexml.apps.case.models import CommCareCase, CommCareCaseAction
+
+from casexml.apps.case.xform import get_case_updates
+from corehq.apps.commtrack.processing import get_stock_actions
+from dimagi.utils.couch.database import iter_docs
+from corehq.apps.tzmigration import set_migration_started, \
+    set_migration_complete
+from corehq.apps.tzmigration.planning import PlanningDB
+from couchforms.dbaccessors import get_form_ids_by_type
+from couchforms import convert_xform_to_json
+from couchforms.models import XFormInstance
+from couchforms.util import adjust_datetimes, scrub_meta
+from corehq.apps.tzmigration import force_phone_timezones_should_be_processed
+
+
+def run_timezone_migration_for_domain(domain):
+    set_migration_started(domain)
+    _run_timezone_migration_for_domain(domain)
+    set_migration_complete(domain)
```

Comment:
should we delete the planning db here? am I wrong to think that we're going to run out of space on the disk pretty quickly if we don't clean up?